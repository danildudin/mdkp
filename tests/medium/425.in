30 7
169 756 854 761 158 75 862 336 706 660 989 587 732 482 193 765 943 352 176 16 737 948 827 506 253 8 880 943 63 81 
905 897 345 443 738 558 74 969 512 442 442 136 473 367 827 377 70 91 638 380 480 684 132 124 621 830 451 16 402 695 9245
386 443 279 883 136 586 108 94 574 601 780 858 579 48 731 685 879 556 705 96 19 772 495 15 345 405 716 423 220 755 9506
227 659 690 335 397 959 977 676 42 166 417 685 171 121 978 740 181 501 256 103 781 570 117 666 844 500 584 894 747 169 5938
947 866 93 622 315 20 514 876 587 306 826 553 514 453 670 199 485 586 709 835 959 596 389 352 50 602 385 658 13 517 9513
990 128 323 741 33 398 724 591 761 526 636 349 989 813 328 695 364 211 906 348 893 614 854 409 339 77 658 341 923 319 9027
842 329 191 687 752 703 356 875 539 189 106 66 165 878 248 912 620 985 141 547 819 651 907 765 980 216 223 916 775 609 5027
707 410 885 934 34 389 353 631 857 739 570 893 453 824 374 840 911 451 770 882 104 136 296 810 688 875 40 409 386 662 6437
