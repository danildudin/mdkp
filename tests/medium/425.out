9828
12
1 2 3 6 8 9 10 12 16 20 21 26 
